app/test/service/openid/github_openid_test.dart

```diff
@@ -49,4 +54,70 @@ void main() {
     expect(firstRsaKey.n, isNotEmpty);
     expect(firstRsaKey.e, isNotEmpty);
   });
+
+  if (Platform.environment['CI'] == 'true') {
```

Comment:
use the tag presubmit only:
https://github.com/dart-lang/pub-dev/blob/master/app/dart_test.yaml#L2